snorkel/labeling/apply/core.py

```diff
@@ -73,20 +73,25 @@ class LFApplier(BaseLFApplier):
     useful for testing.
     """
 
-    def apply(self, data_points: DataPoints) -> np.ndarray:
+    def apply(self, data_points: DataPoints, progress_bar: bool = True) -> np.ndarray:
         """Label list of data points with LFs.
 
         Parameters
         ----------
         data_points
             List of data points to be labeled by LFs
+        progress_bar
+            Display a progress bar?
 
         Returns
         -------
         np.ndarray
             Matrix of labels emitted by LFs
         """
         labels = []
-        for i, x in tqdm(enumerate(data_points)):
+        gen = enumerate(data_points)
+        if progress_bar:
+            gen = tqdm(gen)
+        for i, x in gen:
```

Comment:
Replace this 4-line block with ` for i, x in tqdm(gen, disable=(not progress_bar))`.